cloud/README.md

```diff
@@ -25,61 +25,98 @@ The following clients and Pulsar CLI tools are supported to connect to cluster t
 - pulsar-client
 - pulsar-perf
 
-## How to get token options
+Before starting, we assume that the `service account`, `pulsar instance` and `pulsar cluster` have been created in the current environment, and their `names` and `namespace` as follows:
 
-When you use Token to connect to Pulsar cluster, you need to provide the following options:
+```text
+# service account
+name: test-sa-name
+namespace: test-sa-ns
+
+# pulsar instance
+name: test-pi-name
+namespace: test-pi-ns
+
+# pulsar cluster
+name: test-pc-name
+namespace: test-pc-ns
+```
+
+## How to get service URL
 
 - `SERVICE_URL`
 - `WEB_SERVICE_URL`
-- `AUTH_PARAMS`
 
 For the `SERVICE_URL` field, you can get the **hostname** through the following command:
 
 ```shell script
-$ snctl get pulsarclusters [CLUSTER_NAME] -n [NAMESPACE] -o json | jq '.spec.serviceEndpoints[0].dnsName'
+$ snctl pulsarclusters get test-pc-name -n test-pc-ns -o json | jq '.spec.serviceEndpoints[0].dnsName'
```

Comment:
How about keep both command in this example?
`$ snctl pulsarclusters get test-pc-name -n test-pc-ns -o json | jq '.spec.serviceEndpoints[0].dnsName'`
and 
`$ snctl pulsarclusters get [CLUSTER_NAME] -n [NAMESPACE]  -o json | jq '.spec.serviceEndpoints[0].dnsName'`

you could describe like: "the command format is like: ...; and based on the pre-condition it would be like ..."